src/actions/importData.ts

```diff
@@ -0,0 +1,88 @@
+import SimplePath from '../@types/SimplePath'
+import Thunk from '../@types/Thunk'
+import * as selection from '../device/selection'
+import rootedParentOf from '../selectors/rootedParentOf'
+import isMarkdown from '../util/isMarkdown'
+import strip from '../util/strip'
+import timestamp from '../util/timestamp'
+import { importFilesActionCreator as importFiles } from './importFiles'
+import { importTextActionCreator as importText } from './importText'
+import { newThoughtActionCreator as newThought } from './newThought'
+
+interface ImportDataPayload {
+  path: SimplePath
+  text: string
+  html: string | null
+  rawDestValue: string
+  transient?: boolean
+  isEmText?: boolean
```

Comment:
Parameters should be described with JSDOC comments to make it easier for other developers to understand the use of this action.